From: Hannes Reinecke <hare@suse.de>
Date: Thu, 24 Mar 2016 15:21:18 +0100
Subject: scsi-trace: remove service action definitions
References: bsc#940180,FATE#318723
Patch-Mainline: v4.6
Git-commit: b3bc891eab51c98ec83458e2d7f0db629169ad80

scsi_opcode_name() is displaying the opcode, not the service
action.

Reviewed-by: Ewan D. Milne <emilne@redhat.com>
Signed-off-by: Hannes Reinecke <hare@suse.com>
Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>
Acked-by: Hannes Reinecke <hare@suse.de>
---
 include/trace/events/scsi.h | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/include/trace/events/scsi.h b/include/trace/events/scsi.h
index 079bd10..5c0d91f 100644
--- a/include/trace/events/scsi.h
+++ b/include/trace/events/scsi.h
@@ -95,10 +95,6 @@
 		scsi_opcode_name(VERIFY_16),			\
 		scsi_opcode_name(WRITE_SAME_16),		\
 		scsi_opcode_name(SERVICE_ACTION_IN_16),		\
-		scsi_opcode_name(SAI_READ_CAPACITY_16),		\
-		scsi_opcode_name(SAI_GET_LBA_STATUS),		\
-		scsi_opcode_name(MI_REPORT_TARGET_PGS),		\
-		scsi_opcode_name(MO_SET_TARGET_PGS),		\
 		scsi_opcode_name(READ_32),			\
 		scsi_opcode_name(WRITE_32),			\
 		scsi_opcode_name(WRITE_SAME_32),		\
-- 
1.8.5.6

